mitreattack/navlayers/README.md

```diff
@@ -249,3 +253,21 @@ conf.load_from_file(filename="path/to/poster/config.json")
 t2 = ToSvg(domain='mobile', source='local', local='path/to/local/stix.json', config=conf)
 t2.to_svg(layerInit=lay, filepath="demo2.svg")
 ```
+
+## col_to_md.py
```

Comment:
`col` is more generally understood to mean "column", not "collection." Should probably be written out fully in the file name.

Reply:
Also, this is converting a collection index, not a collection. So even `CollectionToMarkdown` is not technically correct.

Reply:
Gotcha, I'll rename things to hopefully clear things up a bit.